x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -49,22 +49,22 @@ export const PAGE_TITLE = i18n.translate('xpack.securitySolution.detectionEngine
 export const EXPERIMENTAL_ON = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.experimentalOn',
   {
-    defaultMessage: 'Technical preview: On',
+    defaultMessage: 'Advanced sorting capabilities',
   }
 );
 
-export const EXPERIMENTAL_DESCRIPTION = i18n.translate(
-  'xpack.securitySolution.detectionEngine.rules.experimentalDescription',
+export const EXPERIMENTAL_OFF = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.experimentalOff',
   {
-    defaultMessage:
-      'The experimental rules table view is in technical preview and allows for advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.',
+    defaultMessage: 'Advanced sorting capabilities',
   }
 );
 
-export const EXPERIMENTAL_OFF = i18n.translate(
-  'xpack.securitySolution.detectionEngine.rules.experimentalOff',
+export const EXPERIMENTAL_DESCRIPTION = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.experimentalDescription',
   {
-    defaultMessage: 'Technical preview: Off',
+    defaultMessage:
+      'When the setting is on, you can sort the table by all columns. If you experience performance issues when working with the table, you can turn this setting off.',
```

Comment:
Thanks for tagging us in this, @banderror! I reduced the message a bit so there's less text in the bubble. Let me know your thoughts on this revision:

```suggestion
      'Turn this on to enable sorting for all table columns. You can turn it off if you encounter table performance issues.',
```

Reply:
+1 to @nastasha-solomon's trimmed-down tooltip text. Also, would it be possible to shorten the toggle label itself, to "Advanced sorting"? "Capabilities" seems like an unnecessary word, but never mind if the name has already been decided on. Thanks!

Reply:
@nastasha-solomon @joepeeples I don't stop learning from you how to write clear and succinct text in English, thank you for that!